serialization/json_ld/validation.md

```diff
@@ -104,3 +105,75 @@ pyshacl \
 outside of your document, as it cannot understand the use of `import` in
 `SpdxDocument`. For the time being, you will need to manually verify these
 references and ignore the warnings.
+
+## Enable real-time validation during editing
+
+Some code editors offer real-time validation of JSON against a schema as you
+edit. This feature is particularly handy for quickly identifying the location
+of errors or warnings.
+
+### Real-time structural validation in Visual Studio Code
+
+For instance, in Visual Studio Code, you can
+[enable JSON validation](https://code.visualstudio.com/docs/languages/json#_intellisense-and-validation)
+by navigating to `Settings` > `Extensions` and activate the
+`JSON › Validate: Enable` setting by ticking the checkbox.
+
+![Visual Studio Code settings for JSON validation](./validation-vscode.png "A screenshot of Visual Studio Code settings for JSON validation")
+
+Next, edit your `settings.json` file and add the SPDX JSON Schema
+(`https://spdx.org/schema/3.0.0/spdx-json-schema.json`)
+to the `json.schemas` array.
+
+```json
+"json.schemas": [
+  {
+    "fileMatch": [
+      "*.spdx3.json"
+    ],
+    "url": "https://spdx.org/schema/3.0.0/spdx-json-schema.json"
+  }
+]
+```
```

Comment:
[Clause 4: Conformance](https://spdx.github.io/spdx-spec/v2.3/conformance/) in v2.3 suggests naming convention of `*.spdx.json`.

v3.0 has no chapter on conformance yet.

This can be modifed to whatever agreed.